build.gradle

```diff
@@ -89,7 +89,8 @@ nugetSpec {
             summary: 'ILRepack is a utility that can be used to merge multiple .NET assemblies into a single assembly'
         ],
         files: [
-            { file(src: repack.ext.repacked, target: 'tools') }
+            { file(src: repack.ext.repacked, target: 'tools') },
+            { file(src: "..\\..\\..\\ILRepack.props", target: 'build') }
```

Comment:
I've no idea, never used gradle before. The file is at the solution level, so, dunno what 'project' refers to. I know 'new File(".", ...)' didn't work